`SignOutSessionStateManager` is obsolete.

.NET7へ更新するとビルド時に次の警告が表示される。
```cs
warning CS0618: 'SignOutSessionStateManager' は旧形式です ('Use 'Microsoft.AspNe
tCore.Components.Webassembly.Authentication.NavigationManagerExtensions.NavigateToLogout' instead.')
```
対象のプロジェクトは以下。
* blazorwasm-hosted-singleOrg
* blazorwasm-standalone-singleOrg

--- a/src/blazorwasm-hosted-singleOrg/blazorwasm-hosted-singleOrg/Client/Shared/LoginDisplay.razor
+++ b/src/blazorwasm-hosted-singleOrg/blazorwasm-hosted-singleOrg/Client/Shared/LoginDisplay.razor
@@ -2,7 +2,6 @@
 @using Microsoft.AspNetCore.Components.WebAssembly.Authentication
 
 @inject NavigationManager Navigation
-@inject SignOutSessionStateManager SignOutManager
 
 <AuthorizeView>
     <Authorized>
@@ -15,9 +14,8 @@
 </AuthorizeView>
 
 @code{
-    private async Task BeginLogout(MouseEventArgs args)
+    private void BeginLogout(MouseEventArgs args)
     {
-        await SignOutManager.SetSignOutState();
-        Navigation.NavigateTo("authentication/logout");
+        Navigation.NavigateToLogout("authentication/logout");
     }
 }

